plugins/catalog/src/js/repositories/components/RepositoriesPage.js

```diff
@@ -9,7 +10,9 @@ const RepositoriesBreadcrumbs = addButton => {
   const crumbs = [
     <Breadcrumb key={-1} title="Repositories">
       <BreadcrumbTextContent>
-        <Link to="/settings/repositories">Package Repositories</Link>
+        <Link to="/settings/repositories">
+          <Trans>Package Repositories</Trans>
```

Comment:
Should this have a `render="span"`?

Reply:
It seems that the only time I've had to add `render="span"` is when there are unit tests associated because they fail & complain about Invariant Violation

Reply:
You can also avoid the additional span here using:
`<Trans render={<Link to="/settings/repositories" />}>Package Repositories</Trans>`